### DISCUSSION TITLE
Cannot middle click to new tab on same repository links

### DISCUSSION THREAD

**Discussion by SammyCola** [community/community#78904] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When middle clicking on a link, one would expect to open that link in a new tab. If the link is for another file in the same repository, GitHub instead opens the link in the current tab, bypassing the browser's default behavior of middle click to open in a new tab.

**Comment by Kiran1689** [comment]
Hi @UltimateNova1203 

I think its intentional not a bug, when navigating through files in the same repository..
